hooks/command

```diff
@@ -38,13 +43,22 @@ report_coverage() {
   done
 
   wait
+}
+
+report_coverage() {
+  if [ "$report" = false ] ; then
+    return
+  fi
+
+  printf -- "--- :codeclimate: Reporting coverage\\n"
 
   if [[ -v BUILDKITE_PLUGIN_CODECLIMATE_TEST_REPORTER_PARTS ]] ; then
     SUM_PARTS="--parts ${BUILDKITE_PLUGIN_CODECLIMATE_TEST_REPORTER_PARTS}"
   else
     SUM_PARTS=""
   fi
-  ./cc-test-reporter sum-coverage ${debug:+"-d"} ${SUM_PARTS} coverage/codeclimate.*.json
+
+  ./cc-test-reporter sum-coverage ${debug:+"-d"} ${SUM_PARTS} coverage/${formated_file_prefix}.*.json
```

Comment:
```
  ./cc-test-reporter sum-coverage ${debug:+"-d"} "${SUM_PARTS}" coverage/"${formated_file_prefix}".*.json
```
Is recommended if we run this through https://www.shellcheck.net/

Perhaps should be declared as a verification check in there readme

Reply:
I tried this in a variate of ways which all result in the test report command not being able to find the file. I'd assume this might be why  `${SUM_PARTS}` doesn't include double quotes today